lib/puppetserver/ca/utils/http_client.rb

```diff
@@ -96,6 +98,8 @@ def get(url_overide = nil, headers = {})
 
             request = Net::HTTP::Get.new(url.to_uri, headers)
             result = @conn.request(request)
+            
+            @logger.debug("Making a get request at url #{url}")
```

Comment:
Can we move these to before we make the connection? And put the types in all caps? (e.g. `GET`) That's usually how HTTP request types are formatted.

Reply:
Yeah. When you mean put the types in all caps, you mean like "Making a GET request..." right?

Reply:
Yes, and same with the others.

Reply:
Your wish is my command.  It will be capitalized 😄

Reply:
+1 to both of Maggie's suggestions